modules/oadp-operator-supported.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
```

Comment:
Put the table in a module so we can include this wherever you think is relevant

Reply:
:+1: :sweat_smile: As long as nobody asks for xrefs to be added into it because we are not allowed to use xref's in modules.